mailbox/maildir/src/main/java/org/apache/james/mailbox/maildir/MaildirFolder.java

```diff
@@ -344,7 +344,7 @@ public MaildirMessageName getMessageNameByUid(final MailboxSession session, fina
                         int gap = line.indexOf(" ");
                         if (gap == -1) {
                             // there must be some issues in the file if no gap can be found
-                            LOGGER.info("Corrupted entry in uid-file " + uidList + " line " + lineNumber++);
+                            LOGGER.info("Corrupted entry in uid-file {} line {}", uidList, lineNumber++);
```

Comment:
are arguments evaluated _at all_ ? because evaluating lineNumber++ or not make a difference here.

Reply:
Yes, arguments are evaluated. You are right about the side effects making a difference.

Perhaps it would be better to move the increment of `lineNumber` to immediately after the info() call?

Reply:
Actually, it appears that the intent is to increment `lineNumber` with each line read.

I went ahead and pushed a commit to increment `lineNumber` in the "increment" portion of a for loop, both here and a similar place further down in the file.

Reply:
Imo we should have the increment on a separate line before then the call for logging